llvmlite/__init__.py

```diff
@@ -1,3 +1,10 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+# FIXME: Remove me once typed pointers are no longer supported.
+def _get_disable_opaque_pointers():
+  import os
+  return os.environ.get('LLVMLITE_DISABLE_OPAQUE_POINTERS', '0') == '1'
+_disable_opaque_pointers = _get_disable_opaque_pointers()
+del _get_disable_opaque_pointers
```

Comment:
If this global is to be used from outside the module, then it should not be implied to be private by the leading underscore. Also I think the inversion of logic that this then implies elsewhere in the code could be simplified by making it positively named:
```suggestion
def _opaque_pointers_enabled():
  import os
  return os.environ.get('LLVMLITE_DISABLE_OPAQUE_POINTERS', '0') != '1'
opaque_pointers_enabled = _opaque_pointers_enabled()
del _opaque_pointers_enabled
```

Reply:
> If this global is to be used from outside the module, then it should not be implied to be private by the leading underscore

My intention was to keep the global private to the package (`llvmlite`). In other words, I didn't want to introduce a global symbol that other packages could think they could use to enable/disable opaque pointers, as (at the time at least) I thought this should only be done through an environment flag. But I'm happy to change it if you think otherwise.

> I think the inversion of logic that this then implies elsewhere in the code could be simplified

Sure, that sounds good, I'll work on changing this.

Reply:
I've removed the leading underscore and inverted the logic---please let me know what you think!